main.js

```diff
@@ -268,13 +269,17 @@ function gameWin() {
 
   // Create game winning card with players info
   function addWinCard() {
+    setTimeEnd();
+    var timeDifference = gameEnd - gameStart;
+    var seconds = Math.floor(timeDifference / 1000);
+    var minutes = Math.floor(timeDifference / 60000);
```

Comment:
I'm glad we were able to figure this out pretty quickly, performance.now() seems like it'll be easier to work with than Date.now()